src/unix/bsd/netbsdlike/openbsd/mod.rs

```diff
@@ -533,6 +533,12 @@ s! {
         pub key: *mut ::c_char,
         pub data: *mut ::c_void,
     }
+
+    pub struct ifreq {
+        pub ifr_name: [::c_char; ::IFNAMSIZ],
+        #[cfg(libc_union)]
+        pub ifr_ifru: __c_anonymous_ifr_ifru,
+    }
```

Comment:
in not `libc_union` case, the struct size is wrong and could lead to have `ioctl(2)` write outside the struct.